src/main/resources/openapi/use-over-time-1.0.yaml

```diff
@@ -0,0 +1,81 @@
+openapi: 3.0.0
+info:
+  title: Use Over Time
+  version: v1.0
+paths:
+  /eusage-reports/stored-reports/use-over-time:
+    parameters:
+      - $ref: headers/okapi-tenant.yaml
+      - $ref: headers/okapi-token.yaml
+      - $ref: headers/okapi-url.yaml
+      - in: query
+        name: agreementId
+        required: true
+        description: UUID of the agreement we generate the report for
+        schema:
+          type: string
+      - in: query
+        name: startDate
+        required: true
+        description: starting date of use in yyyy-mm-dd format, for example 2020-10-01
+        schema:
+          type: string
+          format: date
+          example: "2020-10-01"
+      - in: query
+        name: endDate
+        required: true
+        description: end date (exclusive) of use in yyyy-mm-dd format, for example endDate=2020-11-01 ends at 2020-10-31T23:59:59.999...
+        schema:
+          type: string
+          format: date
+          example: "2020-11-01"
```

Comment:
Most (if not all) parameters are now defined in parameters sub directory.. Including these.. We need to determine whether we want multiple interfaces (multiple YAML files) or one as PR #36

Reply:
@julianladisch since we prefer to offer one interface (not many), it makes most sense to have ONE yaml file for the whole interface that this module provides. Please use the definitions in master.. But fix the RESPONSE schemas for the 4 last services (as that's clearly NOT correct).

Reply:
I've merged use-over-time-1.0.yaml into eusage-reports-1.0.yaml.
/eusage-reports/stored-reports/use-over-time uses the .yaml from the parameters directory.

I've replaced reportDataEntries.json by report.json as requested by Jakub.

Reply:
Thx